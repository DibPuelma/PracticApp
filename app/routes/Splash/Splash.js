'use strict';

import React, { Component } from 'react'
import { StyleSheet, View, Text, ActivityIndicator } from 'react-native';
import styles from './styles';

export default class Splash extends Component {
  render() {
     return (
      <View style={styles.container}>
        <ActivityIndicator animating={true} style={[styles.centering, {height: 50}]} size="large" color="#3FA9F5"/>
      </View>
    );
  }
}
