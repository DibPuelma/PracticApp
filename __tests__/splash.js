import 'react-native';
import React from 'react';
import Splash from '../app/routes/Splash/Splash'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Splash />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
