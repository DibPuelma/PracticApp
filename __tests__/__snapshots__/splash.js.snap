exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }>
  <ActivityIndicator
    animating={true}
    color="#3FA9F5"
    hidesWhenStopped={true}
    size="large"
    style={
      Array [
        undefined,
        Object {
          "height": 50,
        },
      ]
    } />
</View>
`;
